### PR TITLE
Update Prime FAQ with non-Superheated Rooms in Magmoor info

### PR DESCRIPTION
**Pull request by Samuel6710**
Could be worded a little better

### CODE REVIEW

File: randovania/games/prime1/game_data.py [hunk 1]
```diff
@@ -95,6 +95,12 @@ def _exporter():
             "break fixes present on that version are undone. This reverts the game to be functionally equivalent to "
             "NTSC 0-00, meaning that all versions of Prime are guaranteed to be logically completable when randomized.",
         ),
+        (
+            "Why are some rooms in Magmoor Caverns not superheated?",
+            "Elevator rooms, Storage Cavern, and every room between Twin Fires and the Phazon Mines and "
+            "Phendrana Drifts elevators (Far Magmoor) are not superheated, not including Burning Trail "
+            "and Lake Access which can be encountered without Varia Suit in a vanilla playthrough.",
+        ),
```
Comment (JeffGainsNGames) [inline randovania/games/prime1/game_data.py:103]: You phrased it as a why then didn't say why.

If keeping it this way, you may want to say `In the vanilla game, only certain areas of magmoor are superheated. The far side beyond Twin Fires Tunnel as well as Storage Caverns are exceptions which can be traversed without heat protection.`
I don't know if it's worth mentioning burning trial etc, unless that last sentence makes sense only with the missing reason.

If you rephrase to `What rooms in Magmoor are superheated?`, you could just list the unheated rooms similar to above.

Comment (Samuel6710) [reply randovania/games/prime1/game_data.py:103]: I mainly phrased the question that way since I imagine that would be the general question players would ask. I'll rephrase the answer to work with it better.

Comment (Miepee) [reply randovania/games/prime1/game_data.py:103]: bump?

File: randovania/games/prime1/game_data.py [hunk 2]
```diff
@@ -95,6 +95,11 @@ def _exporter():
             "break fixes present on that version are undone. This reverts the game to be functionally equivalent to "
             "NTSC 0-00, meaning that all versions of Prime are guaranteed to be logically completable when randomized.",
         ),
+        (
+            "What rooms in Magmoor Caverns not superheated?",
```
Comment (Miepee) [inline randovania/games/prime1/game_data.py:99]: TBH i like this question better phrased as something more along the lines of "Why don't I take heat damage in Magmoor Caverns, despite not having varia suit?"

File: randovania/games/prime1/game_data.py [hunk 3]
```diff
@@ -96,9 +96,9 @@ def _exporter():
             "NTSC 0-00, meaning that all versions of Prime are guaranteed to be logically completable when randomized.",
         ),
         (
-            "What rooms in Magmoor Caverns not superheated?",
+            "Why do I not take heat damage in some rooms in Magmoor, despite not having Varia Suit?",
             "Elevator rooms, Storage Cavern, and every room between Twin Fires and the Phazon Mines and "
-            "Phendrana Drifts elevators (Far Magmoor), Burning Trail, and Lake Access.",
+            "Phendrana Drifts elevators (Far Magmoor), Burning Trail, and Lake Access are not coded to be superheated.",
```
Comment (Miepee) [inline randovania/games/prime1/game_data.py:101]: I'd rephrase the answer here a little. Something like "It is vanilla behaviour, that only some rooms are superheated. Here is a list of them: 
- entry1
- entry2"?

**Review by Miepee — CHANGES_REQUESTED** [review]
Needs a changelog entry

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Resolver
 
+### Metroid Prime
+
+- Added: FAQ Entry about non-Superheated rooms
```
Comment (Miepee) [inline CHANGELOG.md:14]: wrong place. youre now suggesting that the seeds entry from below is a prime thing

**Review by Miepee — CHANGES_REQUESTED** [review]

File: randovania/games/prime1/game_data.py [hunk 5]
```diff
@@ -95,6 +95,15 @@ def _exporter():
             "break fixes present on that version are undone. This reverts the game to be functionally equivalent to "
             "NTSC 0-00, meaning that all versions of Prime are guaranteed to be logically completable when randomized.",
         ),
+        (
+            "Why do I not take heat damage in some rooms in Magmoor, despite not having Varia Suit?",
+            "Some rooms in Magmoor are not coded to be superheated. These include: "
+            "• Elevator Rooms "
```
Comment (Miepee) [inline randovania/games/prime1/game_data.py:101]: use minus here instead of unicode

**Review by Miepee — APPROVED** [review]
